fractal_server/app/runner/_slurm/executor.py

```diff
@@ -0,0 +1,228 @@
+# This adapts clusterfutures <https://github.com/sampsyo/clusterfutures>
+# Original Copyright
+# Copyright 2021 Adrian Sampson <asampson@cs.washington.edu>
+# License: MIT
+#
+# Copyright 2022 Jacopo Nepsolo <jacopo.nespolo@exact-lab.it>
```

Comment:
Two issues:
1. this module is a derived work of Sampson's package, which is MIT licensed. We are thus required to keep the original license.
2. I'm starting to prefix the copyright with `#` wherever I see it because otherwise autodoc utilities tend to interpret it as module level documentation.

For these reasons, no change is needed.

Reply:
My point was more that copyright is with FMI/UZH, and we list our names as authors.

> this module is a derived work of Sampson's package, which is MIT licensed. We are thus required to keep the original license.

Sure thing.
BTW, is it enough to mention it here? I guess a note in the main README is in order (copyright is with FMI/UZH, and module X is "Copyright 2021 Adrian Sampson")

Reply:
I think as far as the license goes, it is enough that the README state "copyright by us unless otherwise stated in each individual module". Nevertheless, being nice is a free investment with high returns, so I shall add a comment in the readme.

Reply:
> Nevertheless, being nice is a free investment with high returns

Agreed. I'll also drop them an email, for information, in the next days